processes/processes.md

```diff
@@ -0,0 +1,149 @@
+# Processes
+
+1. [List users processes](#list-users-processes)
+2. [Get a process](#get-a-process)
+3. [Create a process run](#create-a-process-run)
+4. [Get a list of process runs](#get-a-list-of-process-runs)
+
+## List users processes
+
+List all processes available to the user.
+
+```apacheconfig
+GET /api/v1/processes
+```
+
+#### Path parameters
+
+This request does not have any path parameters.
+
+#### Query parameters
+
+| Name | Type |Description |
+| ------------- |------------- | -------------|
+|name|`string`|Filters the results by the process name. When it is not present, returns all the processes that user has access to. |
```

Comment:
@WMKhant Two more filters are added to the API, 1. DateCreateStart and 2. DateCreatedEnd
Please check the API